### PR TITLE
[CRS-49][TS] Revamp landing page of generated templates

### PR DESCRIPTION
**Pull request by biswain2309**
Changes for typescript template. It contains -
* UI changes.
* Inclusion of brand and GitHub link.
* refactoring `assets` folder.
* Removal of redundant files and code. 😮‍💨

It should take care of #49

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,65 @@
...
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.main__brand h1 {
+    margin: 0 0.5rem 0 0;
+    font-size: 2rem;
+}
+
+.main__brand sub {
+    font-size: 1.2rem;
```
Comment (sprakash57) [inline src/index.css:33]: Let's keep it `1rem`.

File: src/App.tsx [hunk 2]
```diff
@@ -24,15 +22,22 @@ const App = () => {
...
+          <sub>v{appInfo.version}</sub>
+        </div>
+        <div>
+          <a className='links card' href="https://github.com/sprakash57/create-react-saga" rel="noopener noreferrer" target="_blank">GitHub</a>
+          <a className='links card' href="https://sprakash57.github.io/create-react-saga" rel="noopener noreferrer" target="_blank">Docs</a>
+        </div>
+      </nav>
+      <img src={wrs} className="main__logo" alt="create-react-saga" />
+      <section>
+        <Button label="Click to run Saga" variant="primary" onClick={() => dispatch(fetchUsers())} data-testid="succss-btn" />
```
Comment (sprakash57) [inline src/App.tsx:38]: Maybe `variant="success"` would be more fitting here.

**Review by sprakash57 — COMMENTED** [review]
Hey, thank you for the changes. Everything looks good. There are two minor issues that can be improved.

File: src/index.css (hunk 1 above)
Comment (biswain2309) [reply src/index.css]: done.

File: src/App.tsx (hunk 2 above)
Comment (biswain2309) [reply src/App.tsx]: done

**Review by sprakash57 — APPROVED** [review]
Nice work.

**Pull request merged**